tests/integration-all/s3/service/core.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function minimal(event, context, callback) {
+  const response = { message: 'Hello from S3! - (minimal)', event };
+  process.stdout.write(event.Records[0].eventSource);
+  process.stdout.write(event.Records[0].eventName);
```

Comment:
Do those writes pay any role? Also there's no whitespace between writes

Reply:
Yes, that's by design. We have to write so that the output shows up in the CloudWatch logs. We could also use `console.log` but I thought that using `process.stdout.write` would be more expressive.

Reply:
Ok, but is it not error-prone, that we output `eventSource` and `eventName` without an interim whitespace? As configured above, in log it'll appear as e.g. `eventSourceeventName`